.gitignore

```diff
@@ -11,3 +11,6 @@ results/
 use-cases/gst_capi/docker-compose_replicas_capi_yolov5.yml
 use-cases/gst_capi/docker-compose_replicas_capi_yolov5_ensemble.yml
 use-cases/gst_capi/docker-compose_replicas_capi_face_detection.yml
+use-cases/demos/docker-compose_replicas_demo_classification.yml
+use-cases/demos/docker-compose_replicas_demo_instance_segmentation.yml
+use-cases/demos/docker-compose_replicas_demo_object_detection.yml
```

Comment:
Now that replica will be standard for the docker compose files we can remove it from the name.

Reply:
@brian-intel  please file this as a separate issue and I will address in a separate PR.